clients/drcachesim/reader/record_file_reader.h

```diff
@@ -177,6 +183,10 @@ class record_reader_t : public std::iterator<std::input_iterator_tag, trace_entr
                     break;
                 }
             }
+            if (cur_entry_.type == TRACE_TYPE_ENCODING)
+                prev_record_was_encoding_ = true;
```

Comment:
nit: prev_record_was_encoding_ = (cur_entry_.type == TRACE_TYPE_ENCODING)

Reply:
Done (or the equivalent with a new helper function for branch target markers too).